pkg/serviceimport/map_test.go

```diff
@@ -163,7 +163,7 @@ var _ = Describe("ServiceImport Map", func() {
 
```

Comment:
I think we should add a test case for non-empty cluster to be complete.

Reply:
Done